apigee.properties

```diff
@@ -2,14 +2,20 @@ category=External Analyzers
 description=Adds XML rules test Apigee apiproxies.
 homepageUrl=https://github.com/CreditMutuelArkea/sonar-apigee-plugin
 archivedVersions=
-publicVersions=3.0.1
+publicVersions=3.0.1,3.0.2
 
 
 defaults.mavenGroupId=com.arkea.satd.sonar.apigee
 defaults.mavenArtifactId=sonar-apigee-plugin
 
+3.0.2.description=Support for SQ 9.7+
+3.0.2.sqVersions=[9.7,LATEST]
+3.0.2.date=2022-12-15
+3.0.2.changelogUrl=https://github.com/CreditMutuelArkea/sonar-apigee-plugin/releases/tag/v3.0.2
+3.0.2.downloadUrl=https://github.com/CreditMutuelArkea/sonar-apigee-plugin/releases/download/v3.0.2/sonar-apigee-plugin-3.0.2.jar
+
 3.0.1.description=Support for SQ 8.8+
-3.0.1.sqVersions=[8.9,LATEST]
+3.0.1.sqVersions=[8.9,9.6.*]
```

Comment:
This range needs to end with the current version (9.7.1 or 9.7.*). Otherwise your users on 9.7.* won't be prompted to do the upgrade.

Reply:
Okay! Thanks for the explanation!

Reply:
You're welcome.
Thank you Ann for the merge. :+1: :gift: